core/controllers/payload_validator.py

```diff
@@ -27,15 +27,41 @@
 from typing import Any, Dict, List, Optional, Tuple, Union
 
 
+def get_schema_type(arg_schema: Dict[str, Any]) -> Any:
+    """This method returns the schema type for an argument by accepting the
+    schema for the corresponding argument.
+
+    Args:
+        arg_schema: dict(str, *). Schema for an argument.
+
+    Returns:
+        str. Returns schema type by extracting it from schema.
+    """
+    return arg_schema['schema']['type']
+
+
+def get_corresponding_key_for_object(arg_schema: Dict[str, Any]) -> Any:
```

Comment:
```suggestion
def get_corresponding_key_for_object(arg_schema: Dict[str, Any]) -> str:
```

Again, please add an explanatory comment.

Reply:
Updated